cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -43,10 +43,26 @@ extern "C" {
  * @brief   Available peripheral buses
  */
 enum {
-    APB1,
-    APB2
+    APB1,           /**< APB1 bus */
+    APB2            /**< APB2 bus */
 };
 
+/**
+ * @brief   Enable the given peripheral clock
+ *
+ * @param[in] bus       bus the peripheral is connected to
+ * @param[in] mask      bit in the RCC enable register
+ */
+void periph_clk_en(uint8_t bus, uint32_t mask);
```

Comment:
no, I think not, as we are not enabling the tree, but only a leaf of it... But I have no strong feelings about this.